share/spack/gitlab/cloud_pipelines/stacks/ml-linux-ppc64le-cpu/spack.yaml

```diff
@@ -0,0 +1,96 @@
+spack:
+  view: false
+  packages:
+    all:
+      require: target=ppc64le
+      variants: ~cuda~rocm
+
+  specs:
+    # Horovod
+    - py-horovod tensor_ops=mpi controllers=mpi frameworks=pytorch
```

Comment:
Are these variants required?

Reply:
Personally, horovod doesn't seem to serve much purpose if it isn't built with MPI.  For the CPU build, NCCL doesn't make sense (does it?), so I stick with MPI.  Everything I've heard is that GLOO isn't all that well supported.  I think a framework is required, so I gave pytorch as it seems to be the most useful/compatible.

Reply:
pytorch is already the default framework and mpi is already the default controller. And `tensor_ops=nccl` should be disabled due to conflicts with `~cuda~rocm`. So I don't think this is needed.

Reply:
Seemingly irrelevant in the face of the fact that py-horovod fails to build even with these default variants as it seems to still depend on py-tensorboard-data-server, which has always failed to build on ppc64le.